sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/administration/DocumentModelAdministrationClient.java

```diff
@@ -77,13 +77,15 @@
 import static com.azure.ai.formrecognizer.documentanalysis.implementation.util.Utility.getTracingContext;
 
 /**
- * This class provides a synchronous client that contains model management the operations that apply
- * to Azure Form Recognizer.
- * Operations allowed by the client are creating, building of custom document analysis models, deleting models,
- * listing models, copying a custom-built model to another Form Recognizer account, composing models from
- * component models, getting operation information and getting resource details.
+ * This class provides a synchronous client to connect with the Form Recognizer Azure Cognitive Service for
+ * building and managing models.
  *
- * <p><strong>Instantiating a synchronous Document Model Administration Client</strong></p>
```

Comment:
The Getting started section and champion scenarios samples can be copied here from Package info.